src/it/ajc_noclasses/invoker.properties

```diff
@@ -0,0 +1 @@
+invoker.goals = clean install
```

Comment:
@amihaiemil at http://plugin.jcabi.com/apidocs-0.13.2/index.html I can see that default phase is `process-classes` and I think we can leave it to speed up the test
BTW, newline is missing at the end of file